src/components/ArticleComponent/View/Layout/MainLayout/index.jsx

```diff
@@ -1,8 +1,12 @@
 import React, { Fragment } from 'react';
 import Article from '../../Content/Article';
+import SocialShare from '../../../../SocialShare';
 
 const MainLayout = () => (
   <div className="ui one centered grid stackable">
+    <div className="two column text-focus-in" style={{ marginTop: '100px' }}>
```

Comment:
This is an inline styling @Oluwaseyi000 , you might want to readjust this using styled component.